gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Tue Oct 03 19:54:39 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
```

Comment:
please use -bin :) -all is a waste of bandwith

Reply:
Sure thing, I changed it to `-bin` - but keep in mind that you're losing out on source support in your IDE when working on Gradle scripts (it can be very helpful). Also - it is a *one-time,  machine-wide* download, normally cached under $HOME/.gradle, so the additional bandwidth requirement only applies *once*. Furthermore, `-all` is the default for new Android Studio projects. Maybe something to consider for the future :)